app/views/comments/_new_comment.mobile.haml

```diff
@@ -6,5 +6,6 @@
   %fieldset
     .control-group
       = hidden_field_tag :post_id,  post_id, :id => "post_id_on_#{post_id}"
-      = text_area_tag :text, nil, :class => "span12 comment_box", :id => "comment_text_on_#{post_id}", :placeholder => t('.comment'), :autofocus => true
-      = submit_tag t('.comment'), :id => "comment_submit_#{post_id}", 'data-disable-with' => t('.commenting'), :class => "btn primary"
+      = text_area_tag :text, nil, :class => "col-md-12 comment_box form-control form-group", :id => "comment_text_on_#{post_id}", :placeholder => t('.comment'), :autofocus => true
```

Comment:
The same here. Overwrite that helper or write a new one which always adds the `form-control` class.